README.md

```diff
@@ -23,13 +24,14 @@ Or
 ```bash
 make start 
 ```
+
 or
 
 ```bash
 make start CONTAINERIZED=true
 ```
 
-This command starts a local development server on http://localhost:3000. Most changes are reflected live without
+This command starts a local development server on <http://localhost:3000>. Most changes are reflected live without
```

Comment:
Let's remove this change. We don't need this URL to be linked.

Reply:
This change is reverted.